linker/Linker/Utils.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace Mono.Linker {
+	public static class Utils {
```

Comment:
Call it TypeNameParser (I hate Utils name)

Reply:
Fixed and noted :smile: